Sources/ArcGISToolkit/Components/FeatureFormView/FormInputs/InputFooter.swift

```diff
@@ -26,6 +26,11 @@ struct InputFooter: View {
     /// The form element the footer belongs to.
     let element: FieldFormElement
     
+    /// An ID regenerated each time the element's value changes.
+    ///
+    /// Allows the footer to be recomputed to reflect changes in validation errors or input length.
+    @State private var id = UUID()
```

Comment:
This is convenient, but I'm wondering if instead of a special-case `@State var` we should be looking at the `.onChanged` event for `element.validationErrors` and triggering off of that. Not sure about how to handle `input length`... thoughts?

Reply:
> I'm wondering if instead of a special-case `@State var` we should be looking at the `.onChanged` event for `element.validationErrors` and triggering off of that.

We'd need to tackle the `Error` equitability issue to watch for new errors with `onChange`. 

Alternatively we could have another `task` awaiting `element.$validationErrors` but we'd also need an additional property to store received errors to make them available to `var primaryError`.  Also, that `task` would *only* help us in collecting the latest validation errors, we'd still need to handle character count changes.

The current changes (`.task { for await _ in element.$value { ...`) helps us keep both the validation errors and the character count refreshed.